src/models.js

```diff
@@ -287,6 +286,18 @@ module.exports = ({ cooler, isPublic }) => {
           })
         );
       });
+      const peersList = await peers;
+      var peersConnected = new Array();
+
+      if (peersList !== undefined) {
+        peersList.forEach(([address, data]) => {
+          // debug(address, data.state);
+          if (data.state === "connected") {
+            peersConnected.push([address, data]);
+          }
+        });
+      }
+      return peersConnected;
```

Comment:
This seems really nasty and would love any recommendations on how to tighten this up.